wondrous-app/components/Common/TaskViewModal/styles.tsx

```diff
@@ -887,3 +891,34 @@ export const TaskIntiativesContainer = styled.div`
   display: flex;
   align-items: center;
 `;
+
+export const AddReviewerButton = styled(ComponentButton)`
+  && {
+    width: 26px;
+    min-height: 0;
+    height: 26px;
+    background: ${palette.grey920};
+    border-radius: 4px;
+    padding: 0;
+    & > button {
+      background: ${palette.grey920};
+      width: 26px;
+      height: 26px;
+      &:hover {
+        background: ${palette.grey920};
+      }
+    }
+  }
+`;
+
+export const ReviewerWrapper = styled.div`
+  width: 100%;
+  flex-wrap: wrap;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 12px;
+  > div:nth-last-child(2) {
```

Comment:
Better and more safe to add a CSS class to the element

Reply:
Why?

Reply:
I agree with adding a class. The reason for this is at some point in the future if we modify the html structure we might miss changing it here

Reply:
How to set the class if the targeted element varies? On the reviewer section, some state updates the second-to-last element.